MonitoringSystem/Client/Types/BaseType.py

```diff
@@ -146,4 +151,30 @@ def addMapping(self, mapping):
 
   ########################################################################
   def getMapping(self):
+    """
+    It returns a specific mapping, which is used by a certain monitoring type 
+    """
     return self.__mapping
+
+  ########################################################################
+  def getPeriod( self ):
+    """
+    
+    It returns the indexing period.
+    
+    """
+    return self.__period
```

Comment:
I want to have private variable. In principle setters/getters can be used for set/get the value of a private variable. Yes, property can be used:
```python
Class A:
  def __init__(self, a ):
    self.__a = a
  
   @property
   def a (self):
     return self.__a
   
   @a.setter
   def a(self, value):
      self.__a = value
```
I can go to this solution. But I do not want to drop the private variables. In that case I only want to change the elastic search index in the base class. Any other place will create a big mess.